src/test/java/com/j256/simplemagic/ContentInfoUtilTest.java

```diff
@@ -40,6 +40,8 @@
 	 */
 	private FileType[] fileTypes = new FileType[] { //
 			//
+			new FileType("/files/x.mkv", ContentType.MATROSKA, "matroska", "video/x-matroska",
```

Comment:
Can you move this to the end of the list?  I'd like to keep these in time order.